core/includes/common.inc

```diff
@@ -3019,14 +3021,13 @@ function backdrop_deliver_html_page($page_callback_result) {
   // domain, but provide a variable to override this. If the code running for
   // this page request already set the X-Frame-Options header earlier, don't
   // overwrite it here.
-  $frame_options = config_get('system.core', 'x_frame_options');
+  $frame_options = config_get('system.core','x_frame_options');
```

Comment:
This change here should be reverted:

```suggestion
  $frame_options = config_get('system.core', 'x_frame_options');
```

Reply:
...also, I believe that the intention was to use:

```suggestion
  $frame_options = $site_config->get('x_frame_options');
```

Reply:
Thanks for the catch, @klonos. Committed.